### ISSUE TITLE
PDF generation doesn't consume notes

### ISSUE THREAD

**Issue by guardrex** [dotnet/docfx#3446]
**DocFX Version Used**: 2.37.2.0

**Template used**: `default`

**Steps to Reproduce**:

Follow PDF guidance at https://dotnet.github.io/docfx/tutorial/walkthrough/walkthrough_generate_pdf.html

**Expected Behavior**:

Notes are consumed and appear in PDF output of topics.

[The PDFs are fine except for this one problem.]

**Actual Behavior**:

Notes aren't used when generating the PDFs.

For ...

```md
> [!NOTE]
> Blazor is an unsupported experimental web framework that shouldn't be used for production workloads at this time.
```

Output ...

<img width="206" alt="capture" src="https://user-images.githubusercontent.com/1622880/45918493-6c16e880-be4d-11e8-8ea0-817367ba63d1.PNG">

Although it looks like "Note" is a link, clicking it only spawns the PDF navigation bar.

**pdf section of docfx.json:**

```json
"pdf": {
  "content": [
    {
      "files": [
        "includes/*.md",
        "community.md",
        "docs/**/*.md",
        "api/**/*.yml",
        "pdf/toc.yml"
      ]
    }
  ],
  "dest": "_site_pdf"
}
```

I attempted to use a `resources` section, too, but it didn't work.

cc: @danroth27 @jongalloway

**Comment by guardrex** [comment]
BTW ... our PR for adding PDF output is over here if you would like to see it :point_right: https://github.com/aspnet/Blazor.Docs/pull/236

**Comment by Blake-Eryx** [comment]
renderAlerts() in docfx.js is getting called on the usual HTML generation (from "build"), but for "pdf" the raw HTML doesn't appear to be going through this step.  Basically, this function adds the classes "alert alert-info" to classes like "NOTE", that is what's missing in the raw HTML being sent to wkhtmlpdf.  From what I can tell, renderAlerts() is called when an HTML file is refreshed--because this raw HTML is being sent straight to wkhtmlpdf, it is never refreshed and these updates never get applied to the output.

**Comment by yufeih** [comment]
Addressed in v2.73.0 with [a new PDF engine](https://dotnet.github.io/docfx/docs/pdf.html).
